pyUltroid/__main__.py

```diff
@@ -18,17 +18,16 @@ def main():
         WasItRestart,
         autopilot,
         customize,
-        fetch_ann,
         plug,
         ready,
         startup_stuff,
     )
     from .startup.loader import load_other_plugins
 
     try:
-        from apscheduler.schedulers.asyncio import AsyncIOScheduler
+        pass
```

Comment:
why try..except pass ?

Reply:
I can re enable this, but I turned the schedulers off to reduce latency and when and if Ultroids fetch ann is still down .

Reply:
and pylint kept doing it disabled now

Reply:
uhm, resolve it

Reply:
wth i did.... :S